src/include/daos_srv/vos.h

```diff
@@ -663,12 +663,13 @@ vos_fetch_begin(daos_handle_t coh, daos_unit_oid_t oid, daos_epoch_t epoch,
  * Finish the fetch operation and release the responding resources.
  *
  * \param ioh	[IN]	The I/O handle created by \a vos_fetch_begin
+ * \param size	[OUT]	The total IO size for the fetch.
```

Comment:
visually misaligned

Reply:
Seems like a GitHub rendering issue. Source code is using tabs and looks correct in a editor.